### PR TITLE
Add xds v3 config & replace docker image repo

### PR DESCRIPTION
**Pull request by champly**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: examples/codes/filter/dubbo/install/Makefile [hunk 1]
```diff
@@ -137,4 +137,4 @@ shell:
 
 proxyv2:
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./build/mosn -ldflags "-X mosn.io/mosn/pkg/types.IstioVersion=${ISTIO_VERSION}" ../../../../../cmd/mosn/main
-	docker build -t symcn.tencentcloudcr.com/symcn/proxyv2:${ISTIO_VERSION} -f ./build/Dockerfile .
+	docker build -t registry.cn-hangzhou.aliyuncs.com/champly/proxyv2:${ISTIO_VERSION} -f ./build/Dockerfile .
```
Comment (wangfakang) [inline examples/codes/filter/dubbo/install/Makefile:140]: It's better to use this url: https://hub.docker.com/repository/docker/mosnio/proxyv2

Comment (champly) [reply examples/codes/filter/dubbo/install/Makefile]: replace with docker.io/mosnio

**Comment by nejisama** [comment]
这个改动以后，能跑原生的1.7.7 + bookinfo吗？之前的是不行的

**Comment by champly** [comment]
> 这个改动以后，能跑原生的1.7.7 + bookinfo吗？之前的是不行的

不行，这个只是针对于 dubbo 的场景做的一个安装文档

**Pull request merged**